src/loaders/stm_load.c

```diff
@@ -270,7 +270,8 @@ static int stm_load(struct module_data *m, HIO_HANDLE * f, const int start)
 	D_(D_INFO "Stored samples: %d", mod->smp);
 
 	for (i = 0; i < mod->ins; i++) {
-		if (mod->xxs[i].len > 1) {
+		if (sfh.ins[i].volume && sfh.ins[i].length) {
```

Comment:
I keep seeing this check for default sample volume everywhere (probably because everyone copies libmikmod/libmodplug) but it's wrong. Any sample should be loaded regardless of its default volume. Yes, even those garbage samples in fracture.stm.